tests/test_smartcar.py

```diff
@@ -52,7 +52,22 @@ def test_get_auth_url(self):
             'client_id': self.client_id,
             'redirect_uri': self.redirect_uri,
             'approval_prompt': 'force',
-            'mock': 'true',
+            'mode': 'test',
+            'scope': ' '.join(self.scope),
+            'state': 'stuff'
+        })
+        expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
+        self.assertEqual(actual, expected)
+
+    def test_get_auth_url_live_mode(self):
```

Comment:
Nit: Looking at the code, it seems that live mode is the default. Can you change the naming of the test above and this test to be `test_get_auth_url` and `test_get_auth_url_test_mode`?